### DISCUSSION TITLE
Prompts to select folder as security scoped bookmark cannot be created

### DISCUSSION THREAD

**Discussion by sebsto** [iterate-ch/cyberduck#16367] [Q&A] [answered]
Hello,

Since I upgraded to macOS Sequoia, Cyberduck 9.0.2 shows 4 file selection dialogs when starting the app, before I can access the S3 browser view. After that, it works as usual.  It will repeatedly show this File open dialog every time I start the app.

See attached animated gif as an example.
![2024-09-22_16-04-46 (1)](https://github.com/user-attachments/assets/55747c55-27ee-452d-8783-5d33fa557668)

Is this a bug ? Is there a workaround ?

**Comment by sebsto** [comment]
Link to Gif : https://github.com/user-attachments/assets/55747c55-27ee-452d-8783-5d33fa557668

**Comment by tatze96** [comment]
Do you use the App Store version of Cyberduck?

Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-16367 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Reply by Xjs** [reply]
I observed the same behaviour and sent a debug log to you, seems to have ended up in your internal ticket CD-12944. Be sure to correlate that one to this issue if you wish to investigate further.

**Reply by sebsto** [reply]
Same problem occurred again today.  I don't why it appears and until when it will interrupt my workflow :-(. I reopened this issue.

**Comment by sebsto** [comment]
@tatze96 
> Do you use the App Store version of Cyberduck
yes 

After a reboot of macOS, I cannot reproduce anymore.

Closing this for now

**Comment by sebsto** [comment]
Problem happened again. I'm re-opening this and I sent my logs files by email (your references CD-12954 and CD-12955)

**Comment by dkocher** [comment, marked as answer]
The prompt is shown [^1] because we fail to obtain a security scoped bookmark for the folder location when running in a sandbox (installed from the Mac App Store). This might be related to a regression in macOS 15 [^2].

[^1]: f98beb4f
[^2]: https://forums.developer.apple.com/forums/thread/764435?answerId=806097022#806097022

**Reply by dkocher** [reply]
I cannot seem to reproduce the issue myself.

**Reply by sebsto** [reply]
Thank you for the investigation. 
from the thread on Apple forums, it appears this will be corrected on macOS 15.1, correct ?

**Reply by dkocher** [reply]
Can you confirm this problem is resolved after upgrading to macOS 15.1?

**Comment by sebsto** [comment]
It's been a while I haven't observed this problem. I can't tell for sure it matches my upgrade to 15.1 
But so far. I haven't been hit since several weeks 
I'm closing this ticket, will reopen if the problem appears again in the future
